pkg/scheduler/framework/plugins/volumebinding/binder.go

```diff
@@ -172,7 +172,7 @@ type SchedulerVolumeBinder interface {
 	// for volumes that still need to be created.
 	//
 	// This function is called by the scheduler VolumeBinding plugin and can be called in parallel
-	FindPodVolumes(pod *v1.Pod, boundClaims, claimsToBind []*v1.PersistentVolumeClaim, node *v1.Node) (podVolumes *PodVolumes, reasons ConflictReasons, err error)
+	FindPodVolumes(pod *v1.Pod, boundClaims, claimsToBind []*v1.PersistentVolumeClaim, unboundVolumesDelayBinding map[string][]*v1.PersistentVolume, node *v1.Node) (podVolumes *PodVolumes, reasons ConflictReasons, err error)
```

Comment:
Change this to accept a PodVolumes type instead of the three parameters (boundClaims, claimsToBind and unboundVolumesDelayBinding)